lib/config/config.js

```diff
@@ -112,16 +109,6 @@ function configLogger(opts = {}) {
 function initialConfig(logger) {
   const cfg = Object.assign({}, confDefaultOptions);
 
-  // Reproduce the generated properties for `Config`.
```

Comment:
Note to reviewer: leftover from the [previous PR](https://github.com/elastic/apm-agent-nodejs/pull/3715). The schema already has values for this internal properties